packages/shared/src/aws/tile.metadata.tileset.ts

```diff
@@ -46,6 +47,15 @@ export class TileMetadataTileSet extends TaggedTileMetadata<TileMetadataSetRecor
         return output;
     }
 
+    /**
+     * Is `rec` a TileSet record
+
+     * @param rec record to infer is a TileMetadataSetRecord
+     */
+    recordIsATileset(rec: BaseDynamoTable): rec is TileMetadataSetRecord {
```

Comment:
Why does this include `A` but the other one doesnt?

I think we also generally use `TileSet`